### PR TITLE
Actually fix color deprecation notices.

### PR DESCRIPTION
**Pull request by zinigor**
Fixes the deprecation notices:

```
PHP Deprecated:  Implicit conversion from float 24.2 to int loses precision in /home/lousy/www/wp-content/plugins/jetpack/_inc/lib/class.color.php on line 105

```

## Proposed changes:
* Fixes deprecation notices for good.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 1]
```diff
@@ -93,16 +93,19 @@ public function fromRgbInt( $red, $green, $blue ) {
 		if ( $red < 0 || $red > 255 ) {
 			throw new RangeException( 'Red value ' . $red . ' out of valid color code range' );
 		}
+		$red = intval( $red );
```
Comment (ice9js) [inline projects/plugins/jetpack/_inc/lib/class.color.php]: So... I know it's only the bottom `$this->color =` line that's causing the warning, but how about we just group these together and put them all at the top of the function for improved readability?

**Review by ice9js — CHANGES_REQUESTED** [review]

File: projects/plugins/jetpack/_inc/lib/class.color.php (hunk 1 above)
Comment (zinigor) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: I don't know about the top - I wanted them to work after we have checked the colors for sane values. What about just doing `intval` directly in the color assignment line?

Comment (ice9js) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: Yeah you're right, we should either bunch them up but after all the conditions or yeah, you can just add the `intvals()` in the color assignment line to keep things clean :).

**Review by ice9js — APPROVED** [review]
👍

**Pull request merged**